src/organisms/documentation/index.js

```diff
@@ -0,0 +1,142 @@
+import React, { useState } from "react"
+import styled from "styled-components"
+import { useToggle } from "react-use"
+import { Text, H5 } from "src/components/typography"
+import { Icon } from "src/components/icon"
+import { Button } from "src/components/button"
+import Flex from "src/components/templates/flex"
+import Layer from "src/components/templates/layer"
+
+const Container = styled(Flex).attrs({
+  padding: [6],
+  width: { max: "325px" },
+  background: "mainBackground",
+  gap: 8,
+  column: true,
+  round: true,
+  overflow: { vertical: "auto" },
+})`
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+`
+const HelpButton = styled(Flex).attrs({
+  background: "primary",
+  round: { side: "left", size: 1 },
+  padding: [2, 1],
+  width: "28px",
+  height: "111px",
+  column: true,
+  alignItems: "center",
+})`
+  cursor: pointer;
+`
+
+const Label = styled(Text).attrs({ color: "bright" })`
+  transform: rotate(90deg);
+  white-space: nowrap;
+  width: 0px;
+  height: 20px;
+`
+
+const Section = ({ icon, title, content, url, children, testid }) => (
+  <Flex width="100%" column gap={2}>
+    <Flex gap={2} alignItems="center">
+      <Icon color="text" name={icon} width="18px" height="18px" />
+      <H5 margin={[0]}>{title}</H5>
+    </Flex>
+    <Flex column gap={4} padding={[0, 2]}>
+      <Text>{content}</Text>
+      <Button
+        width="100%"
+        label="Visit Documentation"
+        as="a"
+        target="_blank"
+        href={url}
+        data-testid={testid}
+      />
+      {children}
+    </Flex>
+  </Flex>
+)
+
+const Help = ({
+  documentationUrl = "https://learn.netdata.cloud/",
+  issuesUrl = "https://github.com/netdata/netdata/issues/new",
+  communityUrl = "https://community.netdata.cloud/c/support/13",
+  communitySupportUrl = "https://community.netdata.cloud/",
+}) => {
+  const [isOpen, toggle] = useToggle()
+
+  return (
+    <Flex background="mainBackgroundDisabled" width="100vw" height="100vh">
```

Comment:
I don't know exactly the business, do we need to occur the entire screen? It seems that a `<Fragment` is needed here